Gemfile

```diff
@@ -28,7 +28,7 @@ gem "hiera", *location_for(ENV['HIERA_LOCATION'] || ['>= 3.2.1', '< 4'])
 # PUP-7115 - return to a gem dependency in Puppet 5
 # gem "semantic_puppet", *location_for(ENV['SEMANTIC_PUPPET_LOCATION'] || ['>= 0.1.3', '< 2'])
 # i18n support (gettext-setup and dependencies)
-gem 'gettext-setup', '>= 0.10', '< 1.0', :require => false
+gem 'gettext-setup', '>= 0.10', '< 1.0', :github => 'joshuaspence/gettext-setup-gem', :branch => 'version', :require => false
```

Comment:
This is needed so that I can actually test this change, but ideally puppetlabs/gettext-setup-gem#40 would be merged and a new version released, in which case I can simply bump the minimum version constraint here.